Generate.py

```diff
@@ -378,6 +378,50 @@ def roll_linked_options(weights: dict) -> dict:
     return weights
 
 
+def handle_trigger_math(math_options: Union[dict, string], option_name: str, weights: dict):
+    options = copy.deepcopy(math_options)
+    if not isinstance(options, dict):
+        options = {str(options): 1}
+    new_dict = {}
+    for name, value in options.items():
+        value_list = []
```

Comment:
I don't see how this could ever not crash.
I don't see any `append` or any other way for this list to be non-empty, but there are a bunch of places indexing into the list.

Reply:
I... Have no idea what I was thinking when I wrote this... I guess I need to stop writing code when I am excessively hot. I could have sworn I re-tested this before pushing this change, but obviously not.